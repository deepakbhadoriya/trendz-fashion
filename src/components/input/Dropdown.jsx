import React from 'react';

import styles from './Dropdown.module.scss';

const Dropdown = ({ title, className, datatestid, options = [] }) => {
  return (
    <div className={className}>
      <h3 className="heading18Grey">{title}</h3>
      <select data-test-id={datatestid} className={styles.select}>
        <option value="">Select</option>
        {options.map((item) => (
          <option value={item} key={item}>
            {item}
          </option>
        ))}
      </select>
    </div>
  );
};

export default Dropdown;
