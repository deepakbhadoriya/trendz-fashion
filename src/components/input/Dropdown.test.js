import React from 'react';

import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

import Dropdown from './Dropdown';

const { mount } = Enzyme;
Enzyme.configure({ adapter: new Adapter() });

describe('<Dropdown/>', () => {
  it('render correctly (snapshot)', () => {
    const tree = mount(<Dropdown />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
