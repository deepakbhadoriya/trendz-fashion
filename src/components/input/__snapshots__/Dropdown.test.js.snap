// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dropdown/> render correctly (snapshot) 1`] = `
<Dropdown>
  <div>
    <h3
      className="heading18Grey"
    />
    <select
      className="select"
      id="cars"
      name="cars"
    >
      <option
        value=""
      >
        Select
      </option>
    </select>
  </div>
</Dropdown>
`;
